gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -236,6 +239,117 @@ done_cb (GtkButton *button, GisSummaryPage *page)
     }
 }
 
+static void
+initial_contact_app_signal (GDBusProxy *proxy,
+                            gchar *sender,
+                            gchar *signal_name,
+                            GVariant *parameters,
+                            gpointer user_data)
+{
+  GisSummaryPage *page = user_data;
+  GisSummaryPagePrivate *priv = gis_summary_page_get_instance_private (page);
+  const gchar *object, *property;
+  g_autoptr(GVariant) value;
+
+  if (g_strcmp0 (signal_name, "ObjectNotify") != 0)
+    return;
+
+  g_variant_get (parameters, "(&s&sv)", &object, &property, &value);
```

Comment:
Since you go to the trouble below of type-checking `value`, I think it's worth type-checking `parameters` here. (If it has the wrong type you'll get criticals from deep inside GVariant, and then `object`, `property` (and `value` as pointed out above) will have undefined contents and you'll be sad.